community/contributing/index.rst

```diff
@@ -1,18 +1,45 @@
 Contributing
 ============
 
+Everyone is most welcome to contribute to Godot. Here are some ways in which you can contribute to the project:
+
+- Improving the engine by fixing bugs, coding new features, or refining existing ones.
+- Writing or proofreading documentation. You can contribute both to this reference manual or the code reference.
+- Translating the editor or documentation from English to other languages.
+- Reporting issues or `writing detailed proposals <https://github.com/godotengine/godot-proposals/>`__.
+
+Getting started
+---------------
+
 .. toctree::
    :maxdepth: 1
    :name: toc-community-contributing
 
    ways_to_contribute
+
+Contributing to the engine
+--------------------------
+
+.. toctree::
+   :maxdepth: 1
+   :name: toc-community-contributing-to-the-engine
+
    best_practices_for_engine_contributors
    pr_workflow
    bisecting_regressions
    code_style_guidelines
    bug_triage_guidelines
-   content_guidelines
+
+Writing documentation
+---------------------
```

Comment:
Following up on my general comment for `_doc_updating_the_class_reference`, I think we should have references to "how to contribute to the documentation" link to this section directly instead of a specific subpage.

Similarly, each toctree section could have its own introduction paragraph in this page that explains what is what, so that users can see at a glance what pages they should read for each use case:
- contributing to the online docs (`documentation_guidelines`, `docs_writing_guidelines`, `building_the_manual`)
- contributing to the class reference (`updating_the_class_reference`, `class_reference_writing_guidelines`)
- contributing to the localization (`editor_and_docs_localization`)